### DISCUSSION TITLE
گزارش و درخواست بررسی ردیاب‌ها

### DISCUSSION THREAD

**Discussion by SPRZB** [MasterKia/PersianBlocker#70] [General]
در صورت یافتن ردیاب در هر سایت و یا وجود مورد مشکوک، میتونید اینجا گزارش و یا درخواست بدید تا بررسی بشه.

**Comment by SPRZB** [comment]
لطفا سایت sale.iranecar.com رو بررسی کنید. آیا ردیاب داره؟

**Reply by MasterKia** [reply]
درخواست های سایت رو بررسی کردم خوشبختانه مشکلی نداشت.
فقط به گوگل هم وصل میشه.

**Reply by SPRZB** [reply]
ممنون از شما

**Comment by SPRZB** [comment]
‏etma.ir رو بررسی میکنید؟

**Reply by MasterKia** [reply]
سپاس.
https://github.com/MasterKia/PersianBlocker/commit/d0f8a39286c1db7b6213993f6f38f8942511af83
